core/src/main/java/com/linecorp/armeria/internal/server/FileAggregatedMultipart.java

```diff
@@ -61,15 +63,30 @@ public ListMultimap<String, MultipartFile> files() {
         return files;
     }
 
-    public static CompletableFuture<FileAggregatedMultipart> aggregateMultipart(ServiceRequestContext ctx,
-                                                                                HttpRequest req) {
+    public static boolean shouldHandle(BodyPart bodyPart, List<String> parameters) {
+        final String name = bodyPart.name();
+        assert name != null;
+        if (parameters.isEmpty()) {
+            return true;
```

Comment:
This seems counterintuitive since it means: Just aggregate/write all body parts even if there are no parameters for the `AnnotatedService` method.

Having said this, it seems like this was handled due to https://github.com/line/armeria/pull/5569#discussion_r1645670152 so I guess it's fine to leave as is.